app/mapping/registration/TrustRegistrationModels.scala

```diff
@@ -18,7 +18,8 @@ package mapping.registration
 
 import java.time.LocalDate
 
-import mapping.{DeedOfVariation, TypeOfTrust}
+import mapping.TypeOfTrust
+import models.registration.pages.DeedOfVariation
```

Comment:
You've gone from using the model to pages? Will this affect the value submitted at the point of declaration?